src/plugins/pictureInPicture.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Vencord, a Discord client mod
+ * Copyright (c) 2023 Vendicated and contributors
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+import { Settings } from "@api/Settings";
+import { Devs } from "@utils/constants";
+import definePlugin, { OptionType } from "@utils/types";
+import { React, Tooltip } from "@webpack/common";
+
+export default definePlugin({
+    name: "PictureInPicture",
+    description: "Adds picture in picture to videos (next to the Download button)",
+    authors: [Devs.Lumap],
+    options: {
+        loop: {
+            description: "Whether to make the PiP video loop or not",
+            type: OptionType.BOOLEAN,
+            default: true,
+            restartNeeded: false
+        }
+    },
+    patches: [
+        {
+            find: ".onRemoveAttachment,",
+            replacement: {
+                match: /\.nonMediaAttachment.{0,10}children:\[(\i),/gm,
```

Comment:
```suggestion
                match: /\.nonMediaAttachment.{0,10}children:\[(\i),/,
```
don't add flags unless you need them